leaf-manager.el

```diff
@@ -400,7 +400,8 @@ If FORCE is non-nil, write file if file exist."
                  (yes-or-no-p (format "File exists (%s), overwrite? " leaf-manager-file))))
     (prog1 t
       (with-temp-file leaf-manager-file
-        (insert (leaf-manager--create-contents-string))))))
+        (insert (leaf-manager--create-contents-string))
+        t))))
```

Comment:
Already this Sexp returns `t` via `prog1`, so last `t` is not needed.

Reply:
Reverted this change :)